Labyrinth/src/Physics/Board.java

```diff
@@ -143,6 +147,21 @@ public Tile insert(Tile t, Point location, int direction){
 		return result;
 	}
 	
+	/**
+	 * Finds the furthest tile from the cursorLocation in the direction insertDirection.
+	 * @param insertDirection
+	 * @param cursorLocation
+	 * @return
+	 */
+	public Point getFurthestTile(int insertDirection, Point cursorLocation) {
+		Point newCoords = Directions.move(cursorLocation,insertDirection);
```

Comment:
Can we not make any reference to 'insert'  and 'cursor' in these variable names? 

Right now, these are the arguments being passed in for the one time this method is used, but this method may be used in other unpredictable ways in the future. When writing the method, we should pretend we don't know how it's being used and name variables based on what it knows that it does (locally), so that it is easier for future-us to notice that we can reuse the method.